3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,113 @@
+//Age-ify (A future age calculator)
+
+const yearOfBirth = 1991;
+const yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+
+console.log(`you will be ${age} years old in ${yearFuture}`);
+
+//Goodboy-Oldboy (A dog age calculator)
+
+const dogYearOfBirth = 2017;
+const dogYearFuture = 2027;
+const shouldShowResultInDogYears = true;
+
+let dogAgeInHumanYears = dogYearFuture - dogYearOfBirth;
+let dogAgeInDogYears = dogAgeInHumanYears * 7;
+
+if (shouldShowResultInDogYears) {
+  console.log(
+    `Your dog will be ${dogAgeInDogYears} dog years old in ${dogYearFuture}`
+  );
+} else {
+  console.log(
+    `Your dog will be ${dogAgeInHumanYears} human years old in ${dogYearFuture}`
+  );
+}
+
+//Housey pricey (A house price estimator)
+
+friendsHouse = [
```

Comment:
This one is not declared 😄

Reply:
All changes have been made as per recommendation.